### PR TITLE
split package and added test

### PR DESCRIPTION
**Pull request by mbuerki**

### CODE REVIEW

**Comment by HannesSommer** [comment]
test this please

**Comment by mbuerki** [comment]
test this please

File: catkin_boost_python_test/python/catkin_boost_python_test/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import numpy_eigen
```
Comment (HannesSommer) [inline catkin_boost_python_test/python/catkin_boost_python_test/__init__.py]: rm this.

**Review by HannesSommer — COMMENTED** [review]
LGTM except for the numpy_eigen import and the included binary ( catkin_boost_python_test/test/test.pyc), which should not be under version control.

**Comment by HannesSommer** [comment]
Great. Again it seems to be Trusty that is causing problems :(.

**Review by HannesSommer — APPROVED** [review]
Thanks!

**Pull request merged**